app/src/main/res/layout/item_vendor.xml

```diff
@@ -14,41 +17,47 @@
     <android.support.v7.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_margin="@dimen/dp_3"
+        android:padding="@dimen/dp_3"
+        car_view:cardCornerRadius="@dimen/dp_1"
+        car_view:cardElevation="@dimen/dp_3"
+        tools:cardCornerRadius="@dimen/dp_3"
         >
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:orientation="vertical"
-            android:padding="@dimen/dp_16"
             >
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:gravity="center"
                 android:orientation="horizontal"
                 >
                 <TextView
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:maxLines="2"
-                    android:paddingRight="@dimen/dp_10"
+                    android:padding="@dimen/dp_10"
                     android:text='@{vendor.name}'
                     android:textSize="@dimen/sp_16"
+                    android:textColor="@android:color/black"
+                    android:textStyle="bold"
                     />
                 <ImageView
-                    android:layout_width="@dimen/dp_40"
-                    android:layout_height="@dimen/dp_40"
+                    android:layout_width="@dimen/dp_30"
+                    android:layout_height="@dimen/dp_30"
                     android:onClick="@{() -> viewModel.onEditVendorClick(vendor)}"
-                    android:padding="@dimen/dp_10"
-                    android:src="@drawable/ic_marker"
+                    android:padding="@dimen/dp_3"
+                    android:src="@drawable/ic_pencil_edit_button"
+                    android:layout_marginRight="@dimen/dp_7"
```

Comment:
WARNING: [RtlHardcoded] Consider replacing `android:layout_marginRight` with `android:layout_marginEnd="@dimen/dp_7"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)